internal/value.go

```diff
@@ -244,7 +244,12 @@ func (sv StringValue) ToInt64() (int64, error) {
 	if sv == "" {
 		return 0, nil
 	}
-	return strconv.ParseInt(string(sv), 0, 64)
+	toParse := string(sv)
+	base := 0
+	if !strings.Contains(strings.ToLower(toParse), "0x") {
```

Comment:
should this be starts with? I also might be tempted to invert this if so that base 10 is the default and we set 0 inside of the condition

Reply:
`-0x` and `+0x` are valid, so I'm using contains, if its malformed `ParseInt` will raise an error for the user